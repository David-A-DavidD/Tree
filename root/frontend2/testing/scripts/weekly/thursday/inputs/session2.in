login
admin
addcredit
buyUser
50.00
list
logout
exit