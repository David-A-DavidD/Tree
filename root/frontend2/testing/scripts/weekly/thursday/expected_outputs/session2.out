Enter transaction code (or 'exit' to quit): Enter username: Login successful.
Enter the username: Enter the amount of credit to add: Credit added successfully. New credit for user buyUser: 50.00
Available Games:
Game Name                    Seller           Price
mygame_____________________   admin           100.00
----------------------------------------------------
Mario______________________   admin           100.00
----------------------------------------------------
halo3______________________   admin           100.00
----------------------------------------------------
gameForSale________________   sellUser        100.00
----------------------------------------------------
Logout successful.